File: src/main/java/io/myosotisdev/utopianism/modules/stat/RequiredLevel.java
package io.myosotisdev.utopianism.modules.stat;

import io.myosotisdev.utopianism.modules.stat.data.ItemStatData;
import io.myosotisdev.utopianism.modules.stat.type.IntegerStat;
import net.minestom.server.item.ItemStack;

public class RequiredLevel extends IntegerStat
{
    public RequiredLevel()
    {
        super("required-level");
    }

    @Override
    public void onApply(ItemStack itemStack, ItemStatData data)
    {
        super.onApply(itemStack, data);
    }
}

File: src/main/java/io/myosotisdev/utopianism/modules/stat/RequiredLevel.kt
package io.myosotisdev.utopianism.modules.stat

import io.myosotisdev.utopianism.modules.stat.data.ItemStatData
import io.myosotisdev.utopianism.modules.stat.type.IntegerStat
import net.minestom.server.item.ItemStack

class RequiredLevel : IntegerStat("required-level")
{
    override fun onApply(itemStack: ItemStack, data: ItemStatData<in Int>)
    {
        super.onApply(itemStack, data)
    }
}